frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx

```diff
@@ -61,7 +88,47 @@ export function useInventoriesSourcesToolbarActions(view: IAwxView<InventorySour
         isDanger: true,
         isDisabled: (sources: InventorySource[]) => cannotDeleteResources(sources, t),
       },
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.None,
+        isPinned: true,
+        label: t('Sync all'),
+        onClick: syncAll,
+        isDisabled: () => cannotLaunchInventorySourcesUpdate,
+      },
     ],
-    [t, deleteSources, pageNavigate, params.inventory_type, params.id, canCreateSource]
+    [
+      t,
+      deleteSources,
+      pageNavigate,
+      params.inventory_type,
+      params.id,
+      canCreateSource,
+      cannotLaunchInventorySourcesUpdate,
+      syncAll,
+    ]
   );
 }
+
+function useSyncAll(inventory_id: string, refresh: () => Promise<void>): () => void {
+  const url = awxAPI`/inventories/${inventory_id}/update_inventory_sources/`;
+  const postRequest = usePostRequest();
+  const alertToaster = usePageAlertToaster();
+  const { t } = useTranslation();
+
+  return () => {
+    void (async () => {
+      try {
+        await postRequest(url, {});
+        void refresh();
+      } catch (error) {
+        alertToaster.addAlert({
```

Comment:
I don't see any alerts when the sync fails- when should this be visible?

Reply:
This should be visible only when sync request fails, not the sync operation running as task. I dont see any alerts of failing operation in the old app...